javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,129 @@
+let names = [];
+let todoList = [];
+//Main function of Voice Assistant
+function getReply(command) { 
+    if(command.startsWith('Hello')){
+        getResponse(command);
+    }else if(command.startsWith('What is my')){
+       getName(command);
+    }else if(command.startsWith('Add fishing')){
+        addToDoList(command);
+    }else if(command.startsWith('Add singing')){
+        updateToDoList(command);
+    }else if(command.startsWith('Remove')){
+        removeFromToDoList(command);
+    }else if(command.startsWith('What is on')){
+        getMyToDoList(command);
+    }else if(command.startsWith('What day')){
+        getTodayDate(command);
+    }else if(command.startsWith('What is')){
+        calculateSimpleMath(command);
+
+    }else if(command.startsWith('Set')){
+       console.log('Timer set for 4 minutes');
+        let wordsInCommand = command.split(' ');
+        const minutes = wordsInCommand[4];
+        setTimer(minutes);
+    }
+}
+    
+
+//Greeting with Response:
+function getResponse(command){
+    const wordsInCommand = command.split(' ');
+    const nameInCommand = wordsInCommand[wordsInCommand.length - 1];
+    let newNameFlag = true;
+    for (i = 0; i < names.length; i ++) {
+        if (names[i] === nameInCommand && command.startsWith('Hello')) {
+            console.log('Error');
+            newNameFlag = false;
+        }
+    }
```

Comment:
You can use the `.includes` function to check this easier:
```js
const doesNameExist = names.includes(nameInCommand); // this will be true or false
```

Reply:
Ok